blueprints/lazylibrarian/config.yml

```
@@ -0,0 +1,3 @@
+blueprint:
+  lazylibrarian: lazylibrarian
+    pkgs: git wget openssl readline pcre libnghttp2 python36 py36-setuptools py36-sqlite3 py36-openssl unrar ca_root_nss
```

Can we push python to either 37 or 38, preferably 38? (a bugfix release)
ca_root_nss is included in every blueprint btw, I think openssl also is (or at least should be) will check into that!

```suggestion
    pkgs: git wget openssl readline pcre libnghttp2 python38 py38-setuptools py38-sqlite3 py38-openssl unrar
```